native-stack/README.md

```diff
@@ -516,6 +516,12 @@ Boolean indicating whether to hide the search bar when scrolling.
 
 Defaults to `true`.
 
+#### `removeTitle` (iOS only)
+
+Boolean indicating whether to hide the title.
+
+Defaults to `false`.
```

Comment:
I would go with `hideTitle` prop name, as we with most props use `hideXXX` / `showXXX` convention.